components/FullwidthSlider/src/index.scss

```diff
@@ -1,125 +1,127 @@
 @keyframes denhaag-angled-image-next {
   0% {
-    clip-path: var(--denhaag-fullwidth-slider-animation-start-clip-path);
+    clip-path: var(--denhaag-fullwidth-slider-animation-start-clip-path, polygon(5% 0, 100% 0, 100% 100%, 0% 100%));
   }
 
   55% {
-    clip-path: var(--denhaag-fullwidth-slider-animation-start-clip-path);
+    clip-path: var(--denhaag-fullwidth-slider-animation-start-clip-path, polygon(5% 0, 100% 0, 100% 100%, 0% 100%));
   }
 
   100% {
-    clip-path: var(--denhaag-fullwidth-slider-animation-end-clip-path);
+    clip-path: var(--denhaag-fullwidth-slider-animation-end-clip-path, polygon(0 0, 100% 0, 100% 100%, 0% 100%));
   }
 }
 
 .denhaag-fullwidth-slider {
-  position: var(--denhaag-fullwidth-slider-position);
-  font-family: var(--denhaag-fullwidth-slider-font-family);
-  font-weight: var(--denhaag-fullwidth-slider-font-weight);
-  font-size: var(--denhaag-fullwidth-slider-font-size);
-  color: var(--denhaag-fullwidth-slider-color);
-  overflow: var(--denhaag-fullwidth-slider-overflow);
-  max-width: var(--denhaag-fullwidth-slider-max-width-xs);
-  touch-action: var(--denhaag-fullwidth-slider-touch-action);
-  padding-block-start: var(--denhaag-fullwidth-slider-padding-block-xs);
-  padding-block-end: var(--denhaag-fullwidth-slider-padding-block-xs);
+  position: relative;
+  font-family: var(--denhaag-fullwidth-slider-font-family, var(--denhaag-typography-sans-serif-font-family));
+  font-weight: var(--denhaag-fullwidth-slider-font-weight, var(--denhaag-typography-weight-regular));
+  font-size: var(--denhaag-fullwidth-slider-font-size, var(--denhaag-typography-scale-s-font-size));
+  color: var(--denhaag-fullwidth-slider-color, var(--denhaag-color-grey-4));
+  overflow: clip;
+  max-width: var(--denhaag-fullwidth-slider-max-width, 100%);
+  touch-action: pan-y;
+  padding-block: var(--denhaag-fullwidth-slider-padding-block, 32px);
```

Comment:
Ik twijfel of dit niet een var(--denhaag-space-..) moet zijn als fallback? Aangezien het wel een spacing is :P